contracts/SortitionTree.sol

```diff
@@ -35,7 +35,14 @@ contract SortitionTree {
   // level6 256k
   // level7 2M
   uint256 internal root;
+
+  // A mapping from layer => (index => branch). For example, to access the 6th
+  // (0-index) branch in the 2nd layer (right below the root node), call
+  // branches[2][6]. Mappings are used in place of arrays for efficiency.
   mapping(uint256 => mapping(uint256 => uint256)) internal branches;
+
+  // A mapping from index => leaf, acting as an array. For example, to access
+  // the 42nd (0-index) leaf, call leaves[42].
```

Comment:
If we want the 42nd and it's 0-index, should we call `leaves[41]`?

Reply:
I suppose this is confusing, but I meant "0th, 1st, 2nd, 3rd..., would be called with `leaves[0], leaves[1], leaves[2], leaves[3]". Do you know a more concise way to convey that?

Maybe I just say:

"A 0-index mapping from index => leaf, acting as an array. For example, to access the 42nd leaf, call leaves[41]."

Reply:
The alternative version is clear enough IMO. Let's go with it!

Reply:
[5099eeb](https://github.com/keep-network/sortition-pools/pull/171/commits/5099eeb539c059c87194947f9b618a5c7aa5063d)